build.gradle

```diff
@@ -525,15 +519,11 @@ allprojects {
    * Without this they won't properly depend on the shadowed project.
    */
   if (isEclipse || isIdea) {
-	configurations.all { Configuration configuration ->
-	  dependencies.all { Dependency dep ->
-		if (dep instanceof ProjectDependency) {
-		  if (dep.getTargetConfiguration() == 'shadow') {
-			configuration.dependencies.add(project.dependencies.project(path: dep.dependencyProject.path, configuration: 'runtime'))
-		  }
-		}
-	  }
-	}
+    project.plugins.withType(ShadowPlugin).whenPluginAdded {
+      project.afterEvaluate {
```

Comment:
Can this be done when the bundle configuration is added, in BuildPlugin, instead of in an afterEvaluate?

Reply:
BuildPlugin doesn't have `isEclipse` or `isIdea` but I *can* do this after the BuildPlugin is added.

Reply:
> BuildPlugin doesn't have isEclipse or isIdea

Maybe it should? It seems odd to have to do this extra work, it should really be part of the base setup.

Reply:
I'll use it if it is defined.